### ISSUE TITLE
Description field needs a max length. Could be renamed to "Application title" or "short description".

### ISSUE THREAD

**Issue by hukka** [CSCfi/rems#1359]

**Comment by okahilak** [comment]
- [ ] Testi

**Comment by okahilak** [comment]
Decided with Esko to not make a non-straightforward change to the 'Application title' field (set a default for it). I think making it mandatory goes to the same category: the current infrastructure doesn't support it very well.

Changing 'description' to 'title' in the code can be a remaining clean-up step, for which I'll create a new issue. I think it's good to do it only once we have enough experience with the new term 'title'.

Therefore, closing this issue.
